tests/driver_dsp0401/main.c

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2017 Inria
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the DSP0401 4 digit display.
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <inttypes.h>
+
+#include "dsp0401_params.h"
+#include "dsp0401.h"
+#include "xtimer.h"
+
+#define TEST_DELAY      (2u)                    /* 2 seconds delay between each test */
+#define WORD_DELAY      (750 * 1000u)           /* 500 milliseconds delay between each word*/
+
+int main(void)
+{
+    dsp0401_t dev;
+    int result;
+
+    puts("DSP0401 test application\n");
+
+    printf("+------------Initializing------------+\n");
+    result = dsp0401_init(&dev, &dsp0401_params[0]);
+    if (result == -DSP0401_ERR_GPIO) {
+        puts("[Error] Cannot initialize GPIO.");
+        return 1;
+    }
+
+    if (result == -DSP0401_ERR_PWM) {
+        puts("[Error] Cannot initialize PWM.");
+        return 1;
+    }
+
+    printf("Initialization successful\n\n");
+
+    printf("\n+--------Starting Display--------+\n");
```

Comment:
use `puts` here and above, but also consider removing superfluous comments that only bloat binary size